assets/sass/components/adsense/_googlesitekit-adsense-ad-blocking-recovery.scss

```diff
@@ -89,6 +89,10 @@
 		.mdc-button {
 			margin-right: 10px;
 		}
+
+		.googlesitekit-cta-link {
+			color: $c-neutral-n-500;
+		}
```

Comment:
The grey text of the "Create message" link button here: https://github.com/google/site-kit-wp/issues/7338#issuecomment-2157319402 seems like a mistake/oversight.

But either way, unless there's a reference to a Figma mockup showing this button needing to be grey, I don't think we should be doing it. It's already not very clear that it's an interactive element with the lack of underline/button styling, but when colouring it grey it just looks like text. Colour alone isn't a great indicator of interactivity, but without it there's really _no_ indicator.

I don't think we should be including this… frankly even if it's in a mock-up I think we should re-think it because it decreases general usability/discoverability, and accessibility.

```suggestion
```

Reply:
Thanks @tofumatt , make sense. I removed the styling